File: 02_Programming_Fundamentals/04_Programming_Fundamentals_PHP/07_Arrays_Lab/07_Equal_Arrays.php
<?php

$arrFirst = explode(" ", readline());
$arrSecond = explode(" ", readline());
$sum = 0;
$equalArr = true;

for ($i = 0; $i < count($arrFirst); $i++) {
 
 if ($arrFirst[$i] == $arrSecond[$i]) {
   $sum += $arrFirst[$i]; 
 } else {
   echo "Arrays are not identical. Found difference at $i index";
   $equalArr = false;
   break;
 }

}

if ($equalArr == true) {
  echo "Arrays are identical. Sum: $sum";
}

?>

File: 02_Programming_Fundamentals/02_Programming_Fundamentals_with_C#/07_Arrays_Lab/07_EqualArrays.cs
using System;
using System.Linq;

class MainClass {
  public static void Main (string[] args) {
    
    int[] arr1 = Console.ReadLine().Split(" ").Select(int.Parse).ToArray();
    int[] arr2 = Console.ReadLine().Split(" ").Select(int.Parse).ToArray();

    int arr1sum = 0;
    int arr2sum = 0;
    bool checkIdentical = true;

    for (int i = 0; i < arr1.Length; i++) {

      arr1sum += arr1[i];
      arr2sum += arr2[i];

      if (arr1[i] != arr2[i]) {
        Console.WriteLine($"Arrays are not identical. Found difference at {i} index");
        checkIdentical = false;
        break;
      }

    }

    if (checkIdentical) {
      Console.WriteLine($"Arrays are identical. Sum: {arr1sum}");    
    }

  }
}